js/script.js

```diff
@@ -1,205 +0,0 @@
-const startForm = {
-  // UI DECLARATION
-  ui: {
-    form: document.querySelector(".form"),
-    fields: document.querySelectorAll(".input-field"),
-    inputs: {
```

Comment:
check all the inputs, if you can have just one of each, why use classes instead of ids?

Reply:
Can you explain your point?

Whats the matter with classes instead of id's?

Reply:
classes are meant to be reusable and ids are not. You may have troubles with your logic if you use these classes somewhere else in the future